api/handlers/exporter.go

```diff
@@ -6,18 +6,17 @@ import (
 
 	"github.com/attestantio/go-eth2-client/spec/phase0"
 	spectypes "github.com/ssvlabs/ssv-spec/types"
+	"github.com/ssvlabs/ssv/networkconfig"
+	qbftstorage "github.com/ssvlabs/ssv/protocol/v2/qbft/storage"
 
 	"github.com/ssvlabs/ssv/api"
-	exporterapi "github.com/ssvlabs/ssv/exporter/api"
-	"github.com/ssvlabs/ssv/exporter/convert"
 	ibftstorage "github.com/ssvlabs/ssv/ibft/storage"
-	qbftstorage "github.com/ssvlabs/ssv/protocol/v2/qbft/storage"
-	"github.com/ssvlabs/ssv/utils/casts"
 )
 
 type Exporter struct {
-	DomainType spectypes.DomainType
-	QBFTStores *ibftstorage.QBFTStores
+	DomainType        spectypes.DomainType
+	NetworkConfig     networkconfig.NetworkConfig
```

Comment:
since `DomainType` is included in `NetworkConfig`, its probably not needed as a separate member of the struct.